### PR TITLE
Setting logging level to warning for ES.trace

### PR DESCRIPTION
**Pull request by muzaffaryousaf**
https://openedx.atlassian.net/browse/TNL-2441

### CODE REVIEW

**Comment by muzaffaryousaf** [comment]
Link to [es logging](https://github.com/elastic/elasticsearch-py/blob/fe6e9a7f5c832fd02847b99f3e2cb84292eb660b/docs/index.rst#logging).

**Comment by muzaffaryousaf** [comment]
@feanil Would you mind to give it a look . Thanks.
@symbolist PTAL. Thanks.

**Comment by symbolist** [comment]
This looks like a reasonable change. :+1:

File: notesserver/settings/logger.py [hunk 1]
```diff
@@ -69,8 +68,8 @@ def get_logger_config(log_dir='/var/tmp',
             },
             "elasticsearch.trace": {
                 'handlers': handlers,
-                'level': 'DEBUG',
-                'propagate': False,
+                'level': 'WARNING',
+                'propagate': True,
```
Comment (feanil) [inline notesserver/settings/logger.py]: Why do we need to change the propagation?  What other handlers are in the hierarchy of elasticsearch.trace?

Comment (muzaffaryousaf) [inline notesserver/settings/logger.py]: Sorry, i think we don't need to propagate this to their parents.

**Comment by muzaffaryousaf** [comment]
Closing in favor of this https://github.com/edx/edx-notes-api/pull/45

**Pull request closed without merging**